### PR TITLE
KARAF-6220, add principal info to file and log audit logs

### PR DESCRIPTION
**Pull request by sjhiggs**

### CODE REVIEW

**Comment by grgrzybek** [comment]
Looks good, thanks @sjhiggs. I'd only complain about:
```java
if(principalInfos.size() > 0) {
```
(lack of space between `if` and `(`)

**Comment by sjhiggs** [comment]
thanks @grgrzybek, fixed.

File: jaas/modules/src/test/java/org/apache/karaf/jaas/modules/audit/AbstractAuditLoginModuleTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package org.apache.karaf.jaas.modules.audit;
```
Comment (jbonofre) [inline jaas/modules/src/test/java/org/apache/karaf/jaas/modules/audit/AbstractAuditLoginModuleTest.java]: Please, add the ASF header.

Comment (sjhiggs) [reply jaas/modules/src/test/java/org/apache/karaf/jaas/modules/audit/AbstractAuditLoginModuleTest.java]: done

**Pull request merged**